modules/mobile/app/serializers/mobile/v0/contact_info_serializer.rb

```diff
@@ -5,21 +5,59 @@ module V0
     class ContactInfoSerializer
       include JSONAPI::Serializer
 
+      ADDRESS_KEYS = %i[
+        id
+        address_line1
+        address_line2
+        address_line3
+        address_pou
+        address_type
+        city
+        country_name
```

Comment:
This is not really clear from the diff, but I changed country code to country name because that's what's in the vcrs.